staging/src/k8s.io/apiserver/pkg/storage/watch_cache.go

```diff
@@ -65,8 +65,11 @@ type watchCacheEvent struct {
 // (to serve the event in different List/Watch requests), in the
 // underlying store we are keeping pair (key, object).
```

Comment:
Could you update this comment to add those new fields also?

Reply:
Done.